ui/src/components/ClientScopesTable/index.jsx

```diff
@@ -56,60 +63,55 @@ export default class ClientScopesTable extends Component {
   clients = null;
 
   createSortedClientsConnection = memoize(
-    (clientsConnection, selectedScope) => {
-      const extractClients = pipe(
-        filter(
-          pipe(
-            path(['node', 'expandedScopes']),
-            filter(pipe(toLower, includes(selectedScope), length))
```

Comment:
Problem was that those nested filter-pipes were not filtering properly.
I tried to figure out for a while how to debug ramda currying pipelines and decided that modern javascript provides all the tools to achieve same results in a more friendly way. 🤷 

I think we can also get rid of it at some point maybe